README.md

```diff
@@ -35,8 +35,9 @@ Parameters (specified as one object passed into hot-shots):
 * `prefix`:      What to prefix each stat name with `default: ''`
 * `suffix`:      What to suffix each stat name with `default: ''`
 * `globalize`:   Expose this StatsD instance globally. `default: false`
-* `cacheDns`:    Cache the initial dns lookup to *host* , only used
-  when protocol is `uds`, `default: false`
+* `cacheDns`:    Caches dns lookup to *host* for *dnsTtlMilliseconds*, only used
+  when protocol is `udp`, `default: false`
+* `dnsTtlMilliseconds`: time-to-live of dns lookups in milliseconds, when *cacheDns* is enabled. `default: 60000`
```

Comment:
We don't include the time type in other names, so could this name be changed to be dnsTtl?  Actually, to aid even more in understanding it's a part of cacheDns, how about cacheDnsTtl?

Reply:
Sure, changed it to `cacheDnsTtl`.